docs-v2/integrations/google-security-review.mdx

```diff
@@ -0,0 +1,216 @@
+* Make sure your contact details in the Google Cloud console project are up to date. All communication is sent there!
+* Prepare a [demo video](https://support.google.com/cloud/answer/13464321?hl=en&ref_topic=13460882&sjid=4120114855682251882-EU#:~:text=2.%20App%20functionality%20demonstration%20video) exactly as requested by Google (shows end to end flow, including OAuth flow, etc.)
+* For each scope you request, prepare a [detailed justification](https://support.google.com/cloud/answer/13464321?hl=en&ref_topic=13460882&sjid=4120114855682251882-EU#:~:text=Request%20narrowest%20scopes) which includes:
+    * An explanation why narrower scopes would not work, including specifics on what functionality would not work as intended. Read Google's sample justification.
+    * This and the demo video are the most frequent sources of back-and-forth communication.
+
+### Additional prep for CASA Tier 2 assessment
+
+CASA assessments have different tiers, but the only relevant one for you here is Tier 2.
+
+#### Run a security pre-scan
+We recommend running a first security scan before you start your CASA assessment. This lets you fix obvious issues now, without the additional back-and-forth hassle with your CASA vendor.
+
+CASA recommends [Fluidattacks](https://fluidattacks.com/) for static security scans. There is a free, open-source CLI that spins up a pre-configured docker container to run a static (SAST) scan against your codebase. [Instructions are here](https://appdefensealliance.dev/casa/tier-2/ast-guide/static-scan).
+
+Ideally, aim for a clean output, but clear at least high and medium threats. Otherwise you will have at least 2 scan cycles during your actual assessment.
+
+#### Pick a CASA assessment vendor
+Your vendor must be listed on the [CASA Authorized Assessors](https://appdefensealliance.dev/casa/casa-assessors) list.
+
+We have compiled a list of the most common vendors:
+
+**[TAC Security](https://tacsecurity.com/google-casa-cloud-application-security-assessment/) (recommended)** - $540 - $720  
+$540 gets you two revalidation. $720 will get you unlimited revalidations. Sometimes runs 10% discounts, check their website.
+
+A validation is a single scan and output of issues. If you do a pre-scan and fix issues first, you’ll probably only need the two validations to fix remaining issues. Try to avoid scanning again without a fix or an accepted omission for everything picked up in the first scan.
+
+Many Nango customers have used them. They are affordable and have mostly automated the process. This makes them not the best at answering emails and dealing with human interactions. We heard stories where they drop the ball on emails from Google which has caused days of delay. We recommend following up with them daily and reminding them of all communication that requires their attention.
```

Comment:
> This makes them not the best at answering emails and dealing with human interactions. We heard stories where they drop the ball on emails from Google which has caused days of delay.

This seems very anecdotal for a public review of this company – of sorts. I think just mentioning that it is wise to follow up with them daily is enough instead of this.